src/draw/vg_lite/lv_draw_vg_lite.c

```diff
@@ -103,6 +103,10 @@ static void draw_execute(lv_draw_vg_lite_unit_t * u)
     lv_layer_t * layer = u->base_unit.target_layer;
 
     lv_vg_lite_buffer_from_draw_buf(&u->target_buffer, layer->draw_buf);
+    if(!lv_draw_buf_has_flag(layer->draw_buf, LV_IMAGE_FLAGS_PREMULTIPLIED)) {
+        LV_LOG_WARN("vg-lite outputs premultiplied image");
+        lv_draw_buf_set_flag(layer->draw_buf, LV_IMAGE_FLAGS_PREMULTIPLIED);
+    }
```

Comment:
```suggestion
    lv_draw_buf_set_flag(layer->draw_buf, LV_IMAGE_FLAGS_PREMULTIPLIED);
```

Reply:
It's normally when user created a draw buffer and set to canvas. GPU will draw to this buffer with premultiplied RGB. Set the flag silently.